src/rgw/rgw_op.cc

```diff
@@ -6578,6 +6578,11 @@ void RGWListMultipart::execute(optional_yield y)
 
   rgw::sal::Attrs attrs;
   op_ret = upload->get_info(this, s->yield, s->obj_ctx, &placement, &attrs);
+  if (op_ret < 0) {
+    ldpp_dout(this, 0) << "Multipart part listing: Failed to get multipart info. rc=" << op_ret << dendl;
```

Comment:
Use `__func__` (see examples in other places).

Reply:
> Use `__func__` (see examples in other places).

ok

Reply:
Also, I'll discard changes from this source code file, as it is Ceph common code file. Will do changes in sal motr layer to return -ERR_NO_SUCH_UPLOAD.